src/components/shared/Menu.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { screen, render } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import Menu from "./Menu";
+import { MenuProps } from "../../models/MenuProps";
+
+describe("Menu Component", () => {
+  test("if component renders header, card, title, but not children (less than 3 cards)", () => {
+    const mockProps: MenuProps = {
```

Comment:
I would try and avoid calling things mockBlahBlah unless the thing is a mock function. These are real props if you think about it.

Reply:
Updated that now, thanks